src/compose.ts

```diff
@@ -728,8 +726,8 @@ export const composeHandler = ({
 
 											let temp
 
-											if(memory === -1) temp = url.slice(start).replace(/\\+|%20/g, ' ')
-											else temp = url.slice(start, memory).replace(/\\+|%20/g, ' ')
+											if(memory === -1) temp = url.slice(start)
+											else temp = url.slice(start, memory)
```

Comment:
We don't need to replace `%20` with space because we are already calling `decodeURIComponent` earlier.